commons/stream/aggregator/aggregate_list.avsc

```diff
@@ -0,0 +1,10 @@
+{
+  "namespace": "org.radarcns.stream.aggregator",
+  "type": "record",
+  "name": "AggregateList",
+  "doc": "Result of data aggregation.",
```

Comment:
Maybe a more specific doc for this would help to have a clear idea why we moved to this schema.